coderd/provisionerjobs.go

```diff
@@ -111,7 +111,7 @@ func (api *API) handleAuthAndFetchProvisionerJobs(rw http.ResponseWriter, r *htt
 		ids = p.UUIDs(qp, nil, "ids")
 	}
 	tags := p.JSONStringMap(qp, database.StringMap{}, "tags")
-	initiatorID := p.UUID(qp, uuid.Nil, "initiator_id")
+	initiatorID := p.UUID(qp, uuid.Nil, "initiator")
```

Comment:
Just checking; you're saying this will change in future work? If so, then 👍🏻

Reply:
The name of the parameter will remain "initiator". The valid domain for the param will expand from being a valid uuid to also being the name of any existing user, or the special value "me"